docs/handbook/chapter2/solana-introduction.md

```diff
@@ -0,0 +1,6 @@
+Solana, founded by Anatoly Yakovenko in 2017 with the publication of the Solana Whitepaper, introduced Proof-of-History (PoH), a novel mechanism for synchronizing time across distributed systems without mutual trust.
+
+The main value proposition of Solana is solving the blockchain trilemma, i.e., delivering
+scalability, decentralization, and security without sacrificing any of the three mentioned features.
+
+Solana officially launched its mainnet, still labeled as beta, in March 2020, with its native coin SOL incentivizing validator nodes and mitigating network spam through transaction fee payments.
```

Comment:
avoid using phrases such as "still labeled as beta". It will be harder in the future to do revisions.

Reply:
removed the phrase